Updated Desktop URLs and removed CSP and EE URLs

usha-mandya opened the pull request:
- Removed Get Docker drop-down from top navigation
- Added a link to Get started docs
- Fixed the URL to Docker Products

usha-mandya commented:
@thaJeztah Could you PTAL?

thaJeztah commented on _includes/global-header.html:
```diff
@@ -10,24 +10,18 @@
                         <ul class="primary nav navbar-nav">
                             <li><a href="https://docker.com/why-docker">Why Docker?</a></li>
-                            <li><a href="https://docker.com/get-started">Product</a></li>
+                            <li><a href="https://www.docker.com/products">Products</a></li>
```
Looks like we use `www.` as the canonical URL, so we should probably do a find/replace across the repository

`https://docker.com/` -> `https://www.docker.com/`

usha-mandya replied:
Sure, will fix that.

thaJeztah commented on _includes/global-header.html:
```diff
@@ -10,24 +10,18 @@
                                     <h6 class="dropdown-header">For Servers</h6>
-                                    <li><a href="https://hub.docker.com/editions/enterprise/docker-ee-server-windows">Windows Server</a></li>
-                                    <li><a href="https://hub.docker.com/editions/enterprise/docker-ee-server-centos">CentOS</a></li>
+                                    <li><a href="https://hub.docker.com/editions/community/docker-ce-server-centos">CentOS</a></li>
```
I'm wondering what the goal is of these links. Looking at this one (for example) will link to Docker Hub, which only has a description (and the description is actually outdated), and to _install_, it gives a link back to the documentation (https://docs.docker.com/engine/installation/linux/centos/)

thaJeztah commented on _includes/global-header.html:
```diff
@@ -10,24 +10,18 @@
-                                    <li><a href="https://docker.com/partners/aws">AWS</a></li>
-                                    <li><a href="https://docker.com/partners/microsoft">Azure</a></li>
+                                    <li><a href="https://hub.docker.com/editions/community/docker-ce-desktop-mac">Mac</a></li>
+									<li><a href="https://hub.docker.com/editions/community/docker-ce-desktop-windows">Windows</a></li>
```
Looks like indentation went a bit off here

usha-mandya replied:
Fixed.

usha-mandya commented:
Discussed offline with @thaJeztah. 

Looks like we surface the Get Docker URL in multiple places on the home page. Removed this section and added link to the Get started docs instead.
